README.md

```diff
@@ -1,2 +1,28 @@
-# hw01
-Repository for homework 01
+#My name is Zach Manta
+####*And I know how to Git*
```

Comment:
Looks like this one didn't work as intended. Perhaps try without the asterisks or with one fewer #